mobile.js

```diff
@@ -18,3 +18,114 @@ listItem.forEach((item) => {
     popUp.classList.remove('show');
   });
 });
+
+// cards
+
+const cardsUl = document.getElementById('cards');
+
+const projects = [
```

Comment:
- [ ] OPTIONAL: I suggest to set the fixed property  to the modal so that the users aren't able to scroll the body of the page.